server/src/main/java/org/opensearch/indices/recovery/RecoverySettings.java

```diff
@@ -65,6 +65,20 @@ public class RecoverySettings {
         Property.NodeScope
     );
 
+    public static final Setting<Boolean> INDICES_USE_REPLICATION_INDIVIDUAL_RATE_LIMITER_SETTING = Setting.boolSetting(
+        "indices.replication.use_individual_rate_limiter",
+        false,
+        Property.Dynamic,
+        Property.NodeScope
+    );
+
```

Comment:
We shouldn't require a new settings. The way we can disable by default is to make the rate limiter value set as -1 and falling back to common rate limiter

Reply:
good idea